translate.js

```diff
@@ -60,7 +60,15 @@ function translateJsonCompilerOutput (output) {
     var contractInput = output['contracts'][contract];
 
     var gasEstimates = contractInput['gasEstimates'];
-
+    if (!gasEstimates['creation']) {
+      gasEstimates['creation'] = [null, null];
+    }
+    if (!gasEstimates['internal']) {
+      gasEstimates['internal'] = null;
+    }
+    if (!gasEstimates['external']) {
+      gasEstimates['external'] = null;
```

Comment:
Why does it need to be set to `null`?

Reply:
cause this is how `translateGasEstimates` manage it

Reply:
Oh, probably that shouldn't check for `null` but for `undefined`. Will need to double check.

Reply:
`translateGasEstimates` check for null